easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.2.eb

```diff
@@ -0,0 +1,46 @@
+
+easyblock = 'PackedBinary'
+
+name = 'sratoolkit'
+version = '2.5.2'
+
+homepage = 'http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software'
+description = """The SRA Toolkit, and the source-code SRA System Development Kit (SDK), will allow you to 
+programmatically access data housed within SRA and convert it from the SRA format"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/%(version)s/']
+sources = ['sratoolkit.%(version)s-ubuntu64.tar.gz']
+
+executables = [
```

Comment:
@boegel - Okay, there are two symlinks for every binary. Hmmph. 
Do you have a policy for that sort of thing?

In any case, I've started work on this.